web/client/translations/data.en-US

```diff
@@ -203,12 +203,17 @@
             "backgroundSwitcher": "This is the helptext for the BackgroundSwitcher",
             "layerSwitcher": "This is the helptext for the LayerSwitcher",
             "infoButton": "This is the helptext for the InfoButton",
-            "locateBtn": "This is the helptext for the LocateBtn"
+            "locateBtn": "This is the helptext for the LocateBtn",
+            "snapshot": "This is the helptext for the Snapshot"
         },
         "queryform": {
             "datefield": {
                 "wrong_date_range": "Start date must be lower than end date"
-            }
+            },
+        "snapshot": {
```

Comment:
You have to close this bracket --> 
Locale file broken for (en-US): Unexpected end of input